rfc/system/1855-type-metadata.md

```diff
@@ -0,0 +1,128 @@
+# [RR] Typing Metadata
+
+**Authors:**
+
+- @maximsmol
+- @kennyworkman
+
+## 1 Executive Summary
+
+A small patch to introduce flytekit support for python-native type metadata.
+
+## 2 Motivation
+
+Users of flyte wish to have access to arbitrary parameter metadata specified in
```

Comment:
I'm unclear why does it say "docstring" here when the proposal only includes typing.Annotation as the supported way of passing these annotations.

Reply:
this was lazy copy-and-pasting from the sister RFC (https://github.com/flyteorg/flyte/pull/1856), just pushed a fix